packages/engine-server/src/engineClient.ts

```diff
@@ -372,7 +368,7 @@ export class DendronEngineClient implements DEngineClient {
     if (_.isUndefined(noteClean)) {
       throw new DendronError({ message: "error updating note", payload: resp });
     }
-    await this.refreshNotes([noteClean]);
+    await this.refreshNotesV2([{ note: noteClean, status: "update" }]);
```

Comment:
If we are getting rid of the existing `refreshNotes`, why rename to `V2`?

Reply:
that method already existed, `refreshNotes` was just a wrapper around it. wanted to minimize code changes with introducing this API